.github/workflows/ci.yaml

```diff
@@ -16,24 +16,34 @@ jobs:
         with:
           java-version: 11
 
-      - name: Login to GitHub Container Registry
-        uses: docker/login-action@v1
-        with:
-          registry: ghcr.io
-          username: ${{ github.repository_owner }}
-          password: ${{ secrets.GHCR_TOKEN }}
-
       - name: Build and test
-        run: ./gradlew pullProprietaryTestImages build --scan --no-daemon
+        run: ./gradlew build -x :smoke-tests:test --scan --no-daemon
 
-  build_windows:
-    runs-on: windows-latest
+  smoke-test:
```

Comment:
I suspect that `pr.yaml` should also be changed

Reply:
💯